### DISCUSSION TITLE
v4.47.0

### DISCUSSION THREAD

**Discussion by TheLarkInn** [webpack/webpack#17656] [General]
## New Features
* [Security] - Add support for md4 in Node >=18. by @iclanton in https://github.com/webpack/webpack/pull/17628

## New Contributors
* @iclanton made their first contribution in https://github.com/webpack/webpack/pull/17628

**Full Changelog**: https://github.com/webpack/webpack/compare/v4.46.0...v4.47.0

<hr /><em>This discussion was created from the release <a href='https://github.com/webpack/webpack/releases/tag/v4.47.0'>v4.47.0</a>.</em>

**Comment by alexojegu** [comment]
Version 4.47.0 released as `latest`, is that correct?

```
Package                           Current  Wanted  Latest  Location                             Depended by
webpack                           5.88.2   5.88.2  4.47.0  node_modules/webpack                 my-package
```

https://www.npmjs.com/package/webpack?activeTab=versions
![image](https://github.com/webpack/webpack/assets/63747287/b8ee52fa-8243-4137-88a6-431354d7ba3f)
